workers/unity/Packages/com.improbable.gdk.deploymentlauncher/DeploymentLauncher.cs

```diff
@@ -0,0 +1,334 @@
+                    {
+                        deploymentName + "_sim_workers",
+                        simSnapshotPath,
+                        Path.Combine(ProjectRootPath, simWorkerLaunchJson)
+                    });
+                }
+
+                var processResult = await RedirectedProcess.RunInAsync(DotNetWorkingDirectory, Tools.Common.DotNetBinary, ConstructArguments(arguments), true, true);
+                return processResult.ExitCode != 0;
+            }
+
+            private async Task<bool> TriggerStopDeploymentAsync(string deploymentId)
+            {
+                if (!CheckDependencies())
+                {
+                    return false;
+                }
+
+                var arguments = new List<string>{
+                    "stop",
+                    projectName,
+                    deploymentId
+                };
+                var processResult = await RedirectedProcess.RunInAsync(DotNetWorkingDirectory, Tools.Common.DotNetBinary, ConstructArguments(arguments), false, true);
+                return processResult.ExitCode != 0;
+            }
+
+            private async Task<List<DeploymentInfo>> TriggerListDeploymentsAsync()
+            {
+                if (!CheckDependencies())
+                {
+                    return null;
+                }
+
+                var arguments = new List<string> {
+                    "list",
+                    projectName
+                };
+                var processResult = await RedirectedProcess.RunInAsync(DotNetWorkingDirectory, Tools.Common.DotNetBinary, ConstructArguments(arguments), false, true);
+
+                if (processResult.ExitCode != 0)
+                {
+                    return null;
+                }
+
+                // Get deployments from output.
+                var deploymentList = new List<DeploymentInfo>();
+                foreach (var line in processResult.Stdout)
+                {
+                    Debug.Log("Line: " + line);
+                    var tokens = line.Split(' ');
+                    if (tokens.Length != 2)
+                    {
+                        continue;
+                    }
+                    deploymentList.Add(new DeploymentInfo
+                    {
+                        Id = tokens[0],
+                        Name = tokens[1]
+                    });
+                }
+                return deploymentList;
+            }
+
+            private static string GenerateTempSnapshot()
+            {
+                string snapshotPath = FileUtil.GetUniqueTempPathInProject();
+
+                Snapshot snapshot = new Snapshot();
+                var simulatedPlayerCoordinatorTrigger = FpsEntityTemplates.SimulatedPlayerCoordinatorTrigger();
+                snapshot.AddEntity(simulatedPlayerCoordinatorTrigger);
+                snapshot.WriteToFile(snapshotPath);
+
+                return snapshotPath;
+            }
+
+            private static bool CheckDependencies()
```

Comment:
IIRC, this feels like it's duplicated somewhere else...

Reply:
Done. I moved it to `Improbable.Gdk.Tools.Common`. (Was in `Improbable.Gdk.Tools.DownloadCoreSdk`).